### ISSUE TITLE
Bug found!

### ISSUE THREAD

**Issue by yx0516** [jwohlwend/boltz#80]
When i use the following yaml file, i get the complex structure sucessfully.

sequences:
- protein:
    id: A
    sequence: MNACPHSDTLTIDPMITDLAGETSRLRAAGPLTRIDLLGVPALAVTGHTLARQLLTDTRLVKDINAWSLWQSGTVTRQWPLIGMIDVDRSMFTVDGPEHRRLRIKTTQALTRRRLDALKPTIERYVAELLDDLERAGADGAVVDLKSVFAYPLPMRVISALMGVPSEDQEQLLTWYKAFFSILTPQDERLRVIDEMHGYFTEMVRRKTAEPGDDLTSALIYATDGETPLTEEEVIGNLQALVAAGHETTVSLILTAVRALLSHPEQLRLVRDGEIGWETAIEETLRWDGPVIHLLMRFATEDIDLGDAVIPRGEGVVMSYRAIGRDITVHGADADDFDITRATAARHISFGHGPHICPGAALARLEAAIALPALFTRFPHLHPALPLDQIPNLPVLTQNDLSHFPIHLGR
- dna:
    id: B
    sequence: GGGAATGGCAGTATTCATCCACAATG
- dna:
    id: C
    sequence: CCATTGTGGATGAATACTGCCATTCC
- rna:
    id: D
    sequence: GGGUCUCGCGGAACCGGUGAGUACACCGGAAUCCAGGAAACUGGAUUUGGGCGUGCCCCCGCGAGACC
version: 1

However, when i use only the protein sequence in the yaml file, i got no results with "| WARNING: ran out of memory, skipping batch" 
under the same free GPU card with no other runing process. The yaml file as following:

sequences:
- protein:
    id: A
    sequence: MNACPHSDTLTIDPMITDLAGETSRLRAAGPLTRIDLLGVPALAVTGHTLARQLLTDTRLVKDINAWSLWQSGTVTRQWPLIGMIDVDRSMFTVDGPEHRRLRIKTTQALTRRRLDALKPTIERYVAELLDDLERAGADGAVVDLKSVFAYPLPMRVISALMGVPSEDQEQLLTWYKAFFSILTPQDERLRVIDEMHGYFTEMVRRKTAEPGDDLTSALIYATDGETPLTEEEVIGNLQALVAAGHETTVSLILTAVRALLSHPEQLRLVRDGEIGWETAIEETLRWDGPVIHLLMRFATEDIDLGDAVIPRGEGVVMSYRAIGRDITVHGADADDFDITRATAARHISFGHGPHICPGAALARLEAAIALPALFTRFPHLHPALPLDQIPNLPVLTQNDLSHFPIHLGR
version: 1

**Comment by xinyu-dev** [comment]
Same issue here.

My instance: 1xA10 (24GB)
Input: 

```yaml
version: 1
sequences:
  - protein:
      id: A
      sequence: DIQMTQSPSSLSASVGDRVTITCRASQGIRNYLAWYQQKPGKAPKLLIYAASTLQSGVPSRFSGSGSGTDFTLTISSLQPEDVATYYCQRYNRAPYTFGQGTKVEIK
  - protein:
      id: B
      sequence: EVQLVESGGGLVQPGRSLRLSCAASGFTFDDYAMHWVRQAPGKGLEWVSAITWNSGHIDYADSVEGRFTISRDNAKNSLYLQMNSLRAEDTAVYYCAKVSYLSTASSLDYWGQGTLVTVSS
```
Memory usage is low but it still outputs `| WARNING: ran out of memory, skipping batch` error and the prediction fails

**Comment by xinyu-dev** [comment]
My command is: 
```bash
boltz predict input/adalimumab.yaml --out_dir output --devices 1 --output_format pdb --use_msa_server
```
With version `3.0.0`, it generated the out of memory issue above 

The I uninstalled 3.0.0 and reinstalled 0.2.1, the issue is gone

**Comment by yx0516** [comment]
@jwohlwend  Hi, would you help with this issue? Thanks a lot.

**Comment by jwohlwend** [comment]
We just released v0.3.2 which should address some of these issues. You can update with `pip install boltz -U` When testing, please remove any existing output folder for your input and run again! Please let us know.
